.github/workflows/lint.yaml

```diff
@@ -43,4 +43,6 @@ jobs:
             REGISTRY=$(grep "REGISTRY_URL := " $MAKEFILE | cut -d\  -f3)
             echo dev-tools=${REGISTRY}/${IMAGE}:${VERSION} >> "$GITHUB_OUTPUT"
           fi
-      - run: docker run --rm -v ${{ github.workspace }}:/workspace ${{ steps.variables.outputs.dev-tools }} /usr/local/bin/test_lint.sh
+      - run: docker run --rm -e EXCLUDE_LINT_DIRS -v ${{ github.workspace }}:/workspace ${{ steps.variables.outputs.dev-tools }} /usr/local/bin/test_lint.sh
```

Comment:
This file is managed centrally, please make edits in the main repo
https://github.com/GoogleCloudPlatform/terraform-genai-doc-summarization/blob/0ef4fa368e26c3b2a9f9ecac05f86d237bbb7373/.github/workflows/lint.yaml#L15-L16

Reply:
I will keep this here for now to make lint checks happy. Will create a PR in CFT repo.